modules/appeal/src/main/Appeal.scala

```diff
@@ -90,3 +90,18 @@ case class AppealMsg(
     text: String,
     at: DateTime
 )
+
+object Snooze {
+  import scala.concurrent.duration._
+  sealed abstract class Duration(val value: FiniteDuration, val name: String)
+  object Duration {
+    case object TwentyMinutes extends Duration(20 minutes, "20 minutes")
+    case object OneHour       extends Duration(1 hour, "one hour")
+    case object ThreeHours    extends Duration(3 hours, "three hours")
+    case object OneDay        extends Duration(1 day, "one day")
+    case object NextDeploy    extends Duration(30 days, "until next deploy")
+    val all                = List(TwentyMinutes, OneHour, ThreeHours, OneDay, NextDeploy)
+    def apply(key: String) = all.find(_.toString == key)
+  }
+  private[appeal] case class SnoozeKey(modId: User.ID, appealId: User.ID)
+}
```

Comment:
This is almost the same as https://github.com/ornicar/lila/blob/97a46cfa5d7334904d0ee0698e7822ee9314d052/modules/report/src/main/model.scala#L40 but creating a new module to factor out seemed overkill. Should I extend it from the `report` module?